src/components/modern/TextWithCopy.js

```diff
@@ -40,26 +40,29 @@ type Props = {|
   textToCopy?: string,
   style?: ViewStyleProp,
   iconColor?: string,
+  ensCopy?: boolean,
 |};
 
 const TextWithCopy = ({
   children,
   textToCopy,
   style,
   iconColor,
+  ensCopy,
 }: Props) => {
   const { t } = useTranslation();
 
   const copyToClipboard = () => {
     if (!textToCopy) return;
 
     Clipboard.setString(textToCopy);
-    Toast.show({ message: t('toast.addressCopiedToClipboard'), emoji: 'ok_hand' });
+    const message = ensCopy ? t('toast.ensNameCopiedToClipboard') : t('toast.addressCopiedToClipboard');
+    Toast.show({ message, emoji: 'ok_hand' });
   };
 
   return (
     <TouchableContainer onPress={copyToClipboard} disabled={!textToCopy} hitSlop={hitSlop20} style={style}>
-      {typeof children === 'string' ? <Text>{children}</Text> : children}
+      {typeof children === 'string' ? <Text style={style}>{children}</Text> : children}
```

Comment:
Please create a separate `textStyle` prop for text style application. Current way will duplicate certain styles, e.g padding/ marging applying them both to text & container.

Reply:
Done